service/src/main/java/org/cbioportal/service/exception/PatientNotFoundException.java

```diff
@@ -2,13 +2,23 @@
 
 public class PatientNotFoundException extends Exception {
```

Comment:
In the current system, Patient id is only meaningful in a Study, so 2 different studies can have Patient id `15` referring to different patients. So I realized that a Patient id without Study id doesn't mean anything, so I added Study id in the exception. Because whenever we query a Patient from database, we do so with Study id.